buildkite/resource_pipeline.go

```diff
@@ -628,102 +664,121 @@ func (*pipelineResource) Schema(ctx context.Context, req resource.SchemaRequest,
 							"`fork` will create builds when the GitHub repository is forked.",
 							"`none` will not create any builds based on GitHub activity.",
 							"`trigger_mode`",
-							"`code`",
+							"`none`",
```

Comment:
Believe this shouldn't be changed as there is `none` defined two lines above and `code` is valid

Reply:
This is changing the substitution in this [line](https://github.com/buildkite/terraform-provider-buildkite/blob/9d0e4b3363328ecfcc570b1d937aac5e96b0edc9/buildkite/resource_pipeline.go#L660) that describes what the default `trigger_mode` is in the description.

Reply:
gotcha!
reading without the fully expanded code above 🤦